easybuild/tools/filetools.py

```diff
@@ -1186,12 +1186,18 @@ def guess_patch_level(patched_files, parent_dir):
     return patch_level
 
 
-def apply_patch(patch_file, dest, fn=None, copy=False, level=None, use_git_am=False):
+def apply_patch(patch_file, dest, fn=None, copy=False, level=None, use_git_am=False, use_git=False):
     """
     Apply a patch to source code in directory dest
     - assume unified diff created with "diff -ru old new"
+
+    Raises EasyBuildError on any error and returns True on success
     """
 
+    if use_git_am:
+        _log.deprecated('5.0', "'use_git_am' named argument in apply_patch function has been renamed to 'use_git'")
```

Comment:
@Flamefire Arguments should be the other way around here:

```suggestion
        _log.deprecated("'use_git_am' named argument in apply_patch function has been renamed to 'use_git'", '5.0')